datadog/tests/provider_test.go

```diff
@@ -558,9 +563,16 @@ func testCheckResourceAttrs(name string, checkExists resource.TestCheckFunc, ass
 		if len(assertionPair) > 1 {
 			value = assertionPair[1]
 		}
-		funcs = append(funcs, resource.TestCheckResourceAttr(name, key, value))
-		// Use utility method below, instead of the above one, to print out all state keys/values during test debugging
-		//funcs = append(funcs, CheckResourceAttr(name, key, value))
+
+		// Handle TypeSet attributes
```

Comment:
Workaroud to handle TypeSets since referencing them with hashes are no longer supported in terraform sdkv2